scripts/build-rosetta-stages.sh

```diff
@@ -6,39 +6,44 @@ GITBRANCH=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD)
 TAG=$(echo ${GITBRANCH} | sed 's!/!-!; s!_!-!g')
```

Comment:
Same question on this one (can it be separate?) I also would prefer someone else look at + approve but don't think that needs to delay the release tonight